lib/ace/layer/marker.js

```diff
@@ -90,14 +101,13 @@ var Marker = function(parentEl) {
                 else
                     this.drawMultiLineMarker(html, range, marker.clazz, config);
             } else {
-                if (this.session.$bidiHandler.isBidiRow(range.start.row)) {
-                    this.drawBidiSingleLineMarker(html, range, marker.clazz + " ace_start" + " ace_br15", config);
-                } else {
-                    this.drawSingleLineMarker(html, range, marker.clazz + " ace_start" + " ace_br15", config);
-                }
+                this.drawSingleLineMarker(html, range, marker.clazz + " ace_start" + " ace_br15", config);
             }
         }
-        this.element.innerHTML = html.join("");
+        if (this.i !=-1) {
+            while (this.i < this.element.childElementCount)
+                this.element.lastChild.remove();
```

Comment:
remove is not defined on ie11, should we add it to our polifill similar to https://github.com/c9/core/blob/master/plugins/c9.ide.ui/lib_apf.js#L1502-L1506?